Modules/_sqlite/module.c

```diff
@@ -126,23 +134,23 @@ static PyObject* module_complete(PyObject* self, PyObject* args, PyObject*
     return result;
 }
 
-PyDoc_STRVAR(module_complete_doc,
-"complete_statement(sql)\n\
-\n\
-Checks if a string contains a complete SQL statement. Non-standard.");
+/*[clinic input]
+_sqlite3.enable_shared_cache as pysqlite_enable_shared_cache
 
-static PyObject* module_enable_shared_cache(PyObject* self, PyObject* args, PyObject*
-        kwargs)
+    enable as do_enable: int
```

Comment:
```suggestion
    do_enable: int
```

Reply:
I totally agree with you. The reasoning behind this was to stay coherent with the naming in the current built-in docs (`import sqlite3; help(sqlite3)`)

Reply:
What do you say, @corona10? Can I proceed with `enable` iso. `do_enable`?

Reply:
@erlend-aasland 

AFAIK, the current implementation requires do_enable.
So it should be `do_enable` or `do_enalbe as enable` no?

```
>>> sqlite3.enable_shared_cache(enable=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: function missing required argument 'do_enable' (pos 1)
```

Reply:
I don't understand what `enable` iso means ;)

Reply:
Sorry, _iso._ = _instead of_ :)

Reply:
You're right, we should keep the current naming! I'll fix it right away.